File: micronaut-java-parent-example/rest/src/main/java/io/archimedesfw/maven/micronaut/example/todo/Application.java
package io.archimedesfw.maven.micronaut.example.todo;


import io.micronaut.runtime.Micronaut;

public final class Application {

    private Application() {
        // Utility class, don't create instances
    }

    public static void main(String... args) {
        final var applicationContext = Micronaut.run(Application.class);
    }

}

File: micronaut-kotlin-parent-example/rest/src/main/kotlin/io/archimedesfw/maven/micronaut/example/todo/Application.kt
package io.archimedesfw.maven.micronaut.example.todo


import io.micronaut.runtime.Micronaut

object Application {

    @JvmStatic
    fun main(args: Array<String>) {
        val applicationContext = Micronaut.run(Application::class.java)
    }

}
